rest/admin_api.go

```diff
@@ -818,3 +818,92 @@ func (h *handler) handlePurge() error {
 
 	return nil
 }
+
+// sg-replicate endpoints
+func (h *handler) getReplications() error {
+	replications, err := h.db.SGReplicateMgr.GetReplications()
+	if err != nil {
+		return err
+	}
+	bytes, err := base.JSONMarshal(replications)
+	h.writeRawJSON(bytes)
+	return err
```

Comment:
What do you think about switching to h.writeJSON instead for these handlers?

```suggestion
	h.writeJSON(replications)
	return nil
```

Reply:
I agree.  In fact, makes me wonder whether the existing usages of writeRawJSON should be reviewed/changed.